src/app/api/card/utils/cardType.ts

```
@@ -0,0 +1,36 @@
+import externalCards from '../dataExternalCards.json';
+import internalCards from '../dataInternalCards.json';
+import { InvalidCardTypeError } from './errors';
+
+export type CardType = 'external' | 'internal';
+
+export type CardsData = {
+  municipio_id_sus: string;
```

Biome (1.9.4)

[error] This property name should be in camelCase.



(lint/style/useNamingConvention)